Runme.groovy

```diff
@@ -13,6 +13,66 @@ csv reader to read and parse the csv items
 some logic that can sort according to ranking
 some logic that will create a new csv file with most favourite actor for each unique tv show
  */
+// class FileHandler {
+//     public File write(InputStream stream) {
```

Comment:
@ACohen123 the FileHandler class has 2 functions

* `write()`, that takes a InputStream and returns a File, this works with the `getResouceAsStream` which would not need machine specific paths to be passed.

* `readText()` reads the text of a File object, just as you do in your script.

The class is a utility object meant to help with redundant/duplicate code